docs/source/API/extension_dev.rst

```diff
@@ -154,10 +161,8 @@ the root of the package, and set the keyword argument `entry_points` of
 After installing this package, ASReview is extended with the ``asreview
 example`` subcommand. See ``asreview -h`` for this option.
 
-An example of a subcommand extension is the `Visualization Extension
-<https://github.com/asreview/asreview-visualization>`_
 
-Dataset extensions
+Dataset Extensions
 ------------------
 An extension of the :class:`asreview.datasets.BaseDataSet` class.
 
```

Comment:
```
Creating a dataset extension provides quick and easy access for yourself and
other users to a dataset. This way, starting research on a topic can be as quick
as a single installation command bypassing data preprocessing altogether.

Dataset extensions integrate new datasets for use in ASReview, available from
the plugin window in the setup phase, or via Command Line Interface.
```

Instead of current info on line 169 to 171.

I have to write access so I'm doing a review instead.